File: app/src/main/java/net/xzos/UpgradeAll/server/ServerContainer.java
package net.xzos.UpgradeAll.server;

import net.xzos.UpgradeAll.server.log.LogUtil;
import net.xzos.UpgradeAll.server.app.manager.AppManager;

import org.jetbrains.annotations.Contract;

public class ServerContainer {
    public static class AppServer {
        final private static LogUtil Log = new LogUtil();
        final private static AppManager AppManager = new AppManager();

        @Contract(pure = true)
        public static LogUtil getLog() {
            return Log;
        }

        @Contract(pure = true)
        public static AppManager getAppManager() {
            return AppManager;
        }
    }
}

File: app/src/main/kotlin/net/xzos/upgradeAll/server/ServerContainer.kt
package net.xzos.upgradeAll.server

import net.xzos.upgradeAll.server.log.LogUtil
import net.xzos.upgradeAll.server.update.UpdateManager
import org.jetbrains.annotations.Contract

internal class ServerContainer {
    companion object {
        @get:Contract(pure = true)
        val Log = LogUtil()
    }
}
